tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest<6
```

Comment:
I'm wondering why we would specifically mark the version as <6?  Is there an upcoming change that is known to break, or is it in the effort to not break unexpectedly?

Reply:
I normally cap the version like this, but considering this is dev requirements and not even applied consistently I guess we can drop it.